### DISCUSSION TITLE
A feature that keeps the GPU continuously active.

### DISCUSSION THREAD

**Discussion by hadixcanimm** [seerge/g-helper#2288] [Ideas]
Could you add a feature that keeps the GPU constantly active?

For instance, something like TrayPwrD3.

In devices without mux switch, the GPU going to sleep causes several issues. When my processor (i5 10300h) is not using the GPU, it still operates at a constant 5 watts and occasionally boosts its frequencies unnecessarily, leading to freezes in Windows.

I can address these issues using the TrayPwrD3 program, but I think it would be nice to have such a feature integrated within the Ghalper program locally.

**Comment by seerge** [comment]
@hadixcanimm i can't, but you can set "dynamic graphics" in power plan settings to maximize performance, so it will use gpu most of the time
